modules/sff/module/src/sff.c

```diff
@@ -1013,6 +1024,7 @@ sff_info_to_json(sff_info_t* info, cJSON** cjp)
     cjson_util_add_string_to_object(cj, "vendor", "%-16.16s", info->vendor);
     cjson_util_add_string_to_object(cj, "model", "%-16.16s", info->model);
     cjson_util_add_string_to_object(cj, "serial", "%-16.16s", info->serial);
+    cjson_util_add_string_to_object(cj, "serial", "%-4.4s", info->vendor_rev);
```

Comment:
You are adding a second "serial", this probably was supposed to be "vendor-revision" or so.